### PR TITLE
OC-10870: Removed remaining system.out statements

### PR DESCRIPTION
**Pull request by suthirr-openclinica**

### CODE REVIEW

File: web/src/main/java/org/akaza/openclinica/controller/SystemController.java [hunk 1]
```diff
@@ -106,7 +106,7 @@
 
 		Map<String, String> env = System.getenv();
 		for (String envName : env.keySet()) {
-			System.out.format("%s=%s%n", envName, env.get(envName));
+			logger.info("{} = {}", envName, env.get(envName));
 		}
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/controller/SystemController.java]: delete these logs

File: web/src/main/java/org/akaza/openclinica/control/managestudy/CreateSubStudyServlet.java [hunk 2]
```diff
@@ -850,22 +850,21 @@ protected String getAdminServlet() {
             for (EventDefinitionCRFBean eventDef : eventDefCrfList) {
                 sessionBean = edcsInSession.get(i);
 
-                System.out.println("iter:           " + eventDef.getId() + "--db:    " + eventDef.getSubmissionUrl());
-                System.out.println("edcsInSession:  " + sessionBean.getId() + "--session:" + sessionBean.getSubmissionUrl());
-                System.out.println();
+                logger.info("iter:           {} --db:    {}", eventDef.getId(), eventDef.getSubmissionUrl());
+                logger.info("edcsInSession:  {} --session: {} ", sessionBean.getId(), sessionBean.getSubmissionUrl());
```
Comment (svadla-oc) [inline web/src/main/java/org/akaza/openclinica/control/managestudy/CreateSubStudyServlet.java]: change these to debug as well

File: web/src/main/java/org/akaza/openclinica/controller/SystemController.java (hunk 1 above)
Comment (suthirr-openclinica) [reply web/src/main/java/org/akaza/openclinica/controller/SystemController.java]: Removed the log

File: web/src/main/java/org/akaza/openclinica/control/managestudy/CreateSubStudyServlet.java (hunk 2 above)
Comment (suthirr-openclinica) [reply web/src/main/java/org/akaza/openclinica/control/managestudy/CreateSubStudyServlet.java]: Changed the logs to debug mode

**Pull request closed without merging**